arch.sh

```diff
@@ -21,7 +21,8 @@ verify_enable_platform(){
     verify_gcc_enable "${feature}"
 }
 add_os_flags() {
-    CMAKE_BUILD_COMMAND="${CMAKE_BUILD_COMMAND}"
+    #CMAKE_BUILD_COMMAND="${CMAKE_BUILD_COMMAND}"
```

Comment:
I left it as a documentation of what could be changed there, but I had to comment it out, because shellcheck complained that it was a noop. Since it doesn't currently complain in the CI, I assume it might be because of a version difference between the CI and my machine. Mine is 0.7.2.

Reply:
Thanks for the info, I checked and it IS because of the version difference. I had version 0.7.0 and it passed after uncommenting it, but after checking it with version 0.7.2 it failed.

Reply:
Given this, are you OK with leaving it as is or do you still want a change?

Reply:
I'm okay with this, you can keep it this way in both cases.